recipe/meta.yaml

```diff
@@ -9,8 +9,11 @@ source:
   url: http://apache.osuosl.org/libcloud/apache-libcloud-{{ version }}.tar.bz2
   sha1: fbf558d7e52336042dd9a456dec3a73f4b35e38e
 
+  patches:
+    - certifi.patch
```

Comment:
Could we please add a comment that this is just PR ( https://github.com/apache/libcloud/pull/812 )?